README.md

```diff
@@ -1026,6 +1027,43 @@ datetime                   inExchanges  inTopHoldersTotal  outsideExchanges
 2019-04-11 00:00:00+00:00     7.975260          13.296356          5.321096
 ```
 
+### Token Top Transactions
+
+Top transactions for the token of a given project
+
+For the successful execution of the function, the usage of an external function - get_by_path - is needed. The arguments for it are the result from the execution of the query and a list from two items, the first of which is 'query_0' and the second - the key, to which we need to cut the path, and the returned result is a shortened list of the returned values from the query.
```

Comment:
You should put this in the function's documentation. All implementation details should not be known by the documentation that the users will read

Reply:
Like this: https://www.datacamp.com/community/tutorials/docstrings-python#second-head

Reply:
Did so for the "path_to_data" function, which was "get_by_path" before